acceptance/framework/flags/flags.go

```diff
@@ -142,6 +146,7 @@ func (t *TestFlags) TestConfigFromFlags() *config.TestConfig {
 
 	// if the Version is empty consulVersion will be nil
 	consulVersion, _ := version.NewVersion(t.flagConsulVersion)
+	vaultserverVersion, _ := version.NewVersion(t.flagVaultServerVersion)
```

Comment:
You wouldn't need this anymore as this is what converts the string representation to a version.Version object.